odoo/models.py

```diff
@@ -74,11 +74,18 @@
 regex_pg_name = re.compile(r'^[a-z_][a-z0-9_$]*$', re.I)
 regex_field_agg = re.compile(r'(\w+)(?::(\w+)(?:\((\w+)\))?)?')
 
+regex_aggregate_spec = re.compile(r'(\w+)(?::(\w+))?$')
```

Comment:
IDK if it could lead to SQL injection, but just to be sure
```suggestion
regex_aggregate_spec = re.compile(r'^(\w+)(?::(\w+))?$')
```

Reply:
I use `re.match` then the `^` is already implicitly added.

Reply:
not a fan of that being implicit but ok